packages/solana/pubspec.yaml

```diff
@@ -16,6 +16,7 @@ dependencies:
   freezed_annotation: ^0.14.2
   http: ^0.13.3
   json_annotation: ^4.0.1
+  web_socket_channel: ^2.1.0
```

Comment:
Isn't it better to merge the other PR first? So that the contribution of Chikin developer is not lost.

Reply:
This PR basically includes the changes from the other PR, so we can close the other one afterwards.